informer/integration/project.yaml

```diff
@@ -0,0 +1,4 @@
+version: 1
+
+test:
+  env:
```

Comment:
We do not need this here I think. Can we just drop it though? Does e2e-harness support this?

Reply:
Currently it expects to find the `integration/project.yaml` file https://github.com/giantswarm/e2e-harness/blob/master/pkg/project/project.go#L187, the `env` section can be safely dropped https://github.com/giantswarm/e2e-harness/blob/master/pkg/project/project.go#L200

Reply:
Should we make this optional then as well?

Reply:
:+1: I'll prepare the change.